Lecture3/Task1/product/ProductFactoryDirector.java

```diff
@@ -0,0 +1,44 @@
+package product;
+
+import builder.AcousticGuitarBuilder;
+import builder.ElectricGuitarBuilder;
+import builder.Guitar;
+import builder.GuitarBuilderDirector;
+import exceptions.FailedOrderException;
+import utils.RandomUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProductFactoryDirector {
+
+	public static List<Guitar> createGuitars() throws FailedOrderException {
+		List<Guitar> guitars = new ArrayList<>();
+		GuitarBuilderDirector builderDirector = new GuitarBuilderDirector();
+		if (guitars.size() == 0) {
+			int amount = RandomUtils.getAmount();
+			if (amount == 0) {
+				throw new FailedOrderException("Fail to produce some guitars");
+			}
+			for (int i = 0; i < amount; i++) {
+				switch (RandomUtils.getType()) {
```

Comment:
also as in line 19 - provide list of values in params `RandomUtils.getType(Type.values())`